patches/net/minecraft/world/inventory/CrafterMenu.java.patch

```diff
@@ -0,0 +1,15 @@
+--- a/net/minecraft/world/inventory/CrafterMenu.java
++++ b/net/minecraft/world/inventory/CrafterMenu.java
+@@ -117,9 +_,12 @@
+         if (this.player instanceof ServerPlayer serverplayer) {
+             Level level = serverplayer.level();
+             CraftingInput craftinginput = this.container.asCraftInput();
++            // using the player-free context here since the player won't be usable by the dispense method
```

Comment:
Is this comment right? I see a serverplayer variable still being passed in. Looks the same as CraftingMenu patch